db/migrate/20181029132215_add_estatus_to_todos.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddEstatusToTodos < ActiveRecord::Migration[5.2]
+  def change
+    add_column :todos, :estate, :boolean
```

Comment:
did you mean state/status?

Reply:
that's and old migration; at the end i used status